alphalens/performance.py

```diff
@@ -282,8 +285,11 @@ def mean_return_by_quantile(quantized_factor,
         Standard error of returns by specified quantile.
     """
 
-    demeaned_fr = utils.demean_forward_returns(forward_returns,
+    if demeaned	:
```

Comment:
you see, there is a reason if I wear glasses and that's not enough! ;) (actually I looked at the code on github and the web page doesn't show the white space)